### PR TITLE
Remove PV/PVC finalizers on downgrade from >= 1.10 to < 1.10

### PR DESCRIPTION
**Pull request by liggitt**
Codifies the need to remove storage protection finalizers on downgrade from >= 1.10 to < 1.10

See https://github.com/kubernetes/website/pull/7731 and https://github.com/kubernetes/kubernetes/issues/60764#issuecomment-373803755

### CODE REVIEW

File: test/e2e/lifecycle/cluster_upgrade.go
Comment (rootfs) [inline test/e2e/lifecycle/cluster_upgrade.go]: no pvc namespace?

Comment (liggitt) [reply test/e2e/lifecycle/cluster_upgrade.go]: correct, list across all namespaces

**Comment by verult** [comment]
/assign @krousey 
for downgrade test review

File: test/e2e/lifecycle/cluster_upgrade.go
Comment (krousey) [reply test/e2e/lifecycle/cluster_upgrade.go]: `metav1.NamespaceAll` would be more clear

**Review by krousey — CHANGES_REQUESTED** [review]
Is there an issue created tracking the proper fix? If so, this PR should be tagged there so they know exactly what to rollback when they fix it.

**Comment by liggitt** [comment]
> Is there an issue created tracking the proper fix?

tagged in https://github.com/kubernetes/kubernetes/pull/61282

**Comment by liggitt** [comment]
> metav1.NamespaceAll would be more clear

updated

**Review by krousey — APPROVED** [review]

**Comment by krousey** [comment]
LGTM from me. Leaving it to branch/release team to merge.

**Comment by liggitt** [comment]
@saad-ali @childsb for lgtm

**Comment by jpbetz** [comment]
/hold
Hold all unmerged cherrypicks for cut of patch release on Monday.

**Comment by liggitt** [comment]
discussed in sig-release call

as @krousey [pointed out](https://github.com/kubernetes/kubernetes/issues/60764#issuecomment-373813598), the PVC cleanup failure in the downgrade tests did not block running of the remaining downgrade tests, which means the single failure in the downgrade test (namespace cleanup) is well-understood and not masking other failures.

given that we plan to resolve this finalizer issue in the 1.9 stream (in #61282 and #61316) and would need to revert this as part of that, I think it is better to just close this.

**Pull request closed without merging**
